crates/provider-keyvalue-redis/Cargo.toml

```diff
@@ -10,6 +10,7 @@ repository.workspace = true
 
 [dependencies]
 anyhow = { workspace = true }
+async-nats = { workspace = true, features = ["ring"] }
```

Comment:
This _might_ prove to be a problem with our cross compilations, but I'm not 100% sure. We may just want to do a dry run of the release from this branch to double check